app/views/provider_mailer/application_rejected_by_default.text.erb

```diff
@@ -0,0 +1,15 @@
+Dear <%= @application.provider_user_name %>
+
+# Application rejected automatically  
+
+<%= @application.candidate_name %> submitted an application for <%= @application.course_name %> <%= @application.course_code %> on <%= @application.submitted_at %>. 
+
+You did not respond within 40 working days so we rejected the application on your behalf. 
+
+You can log in to Manage teacher training applications to check the status of your applications:
+
+<%= provider_interface_application_choice_url(application_choice_id: @application.application_choice_id) %>
```

Comment:
FYI: this could be `provider_interface_application_choice_url(@application.application_choice)`, I think you don't have to be explicit about the ID.

Reply:
Resolved in e06f8e6